scripts/CEE/get-kafka-instance-state/metadata.yaml

```diff
@@ -41,6 +41,19 @@ rbac:
             - ""
           resources:
             - "pods/exec"
+        - verbs:
+            - "get"
+            - "list"
+          apiGroups:
+            - "apps"
+          resources:
+            - "statefulsets"
+        - verbs:
```

Comment:
Isn't that redundant / too permissive?
Why do you need this section?

Reply:
thanks, well spotted.  I made a mistake copying back from the rbac yaml test case into the metadata.yml.